app/src/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { useAuth0 } from "@auth0/auth0-react";
+
+import "../styles.css";
+
+export default function Header() {
+  const { logout, user, isAuthenticated } = useAuth0();
+
+  return (
+    <div className="header">
```

Comment:
It might be a good idea to use a more semantic tag ie `<header>` vs `<div className="header">`. Or you seem to use this like a navbar, so  `<nav>`  would work too!

Reply:
Thanks, @carbonsoda!

Reply:
is it strange to say `<nav className="header">` for styling?

Reply:
@LeiaQuesada I think thats fine!
https://react-bootstrap.github.io/components/navbar/

Looking at this bootstrap documentation they do that exact thing 
` <Nav className="mr-auto">`